packages/jobs/lib/crons/autoIdleDemo.ts

```diff
@@ -0,0 +1,128 @@
+import { schedule } from 'node-cron';
+import {
+    CommandToActivityLog,
+    ErrorSourceEnum,
+    ScheduleStatus,
+    SyncClient,
+    SyncCommand,
+    SyncConfig,
+    createActivityLog,
+    createActivityLogMessageAndEnd,
+    db,
+    errorManager,
+    updateSuccess as updateSuccessActivityLog,
+    updateScheduleStatus,
+    isErr
+} from '@nangohq/shared';
+import tracer from 'dd-trace';
+
+export async function cronAutoIdleDemo(): Promise<void> {
+    schedule('1 * * * *', () => {
+        const span = tracer.startSpan('cron.syncs.idleDemo');
+        tracer.scope().activate(span, async () => {
```

Comment:
not sure why you use activate here

Reply:
I was in the impression that it was the way to go when dealing with nested async calls, let me double check 👍🏻

Reply:
yup tested, probably good to know for futur improvements 
1st is without, 2nd is with `activate()`
https://github.com/NangoHQ/nango/assets/1637651/80960583-ca2b-4ee8-b5cf-6e058c678d8b

Reply:
Nice, thanks for the investigation

Reply:
> In most cases, it's not necessary to interact with the scope manager since tracer.trace() activates the span on its scope, and uses the active span on the current scope if available as its parent.
Does this from the doc means that it can be written like this:
```
await tracer.trace('SpanTypes.JOBS_IDLE_DEMO', span => {
    exec();
})
```

Reply:
Didn't work for me but worth investigating